src/menus/autoConnectScreen.cpp

```diff
@@ -92,7 +99,17 @@ void AutoConnectScreen::update(float delta)
             else
                 status_label->setText("Connecting...");
             break;
-        case GameClient::WaitingForPassword: //For now, just disconnect when we found a password protected server.
+        case GameClient::WaitingForPassword:
+            if (!tried_password) {
+                auto password = PreferencesManager::get("autoconnect_password");
+                if (password != "") {
+                    game_client->sendPassword(password.upper());
+                    tried_password = true;
```

Comment:
this is never set to false after this, is that intentional?

Reply:
nope, just me mixing up scopes.